### PR TITLE
August

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: log-base/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # log-base-0.7.2 (2017-06-19)
 * mkBulkLogger now uses a bounded queue to interact with the logger thread.
+* Add 'MFunctor LogT' instance
```
Comment (23Skidoo) [inline log-base/CHANGELOG.md]: This is actually the 0.7.1.1 section, the version number is wrong.

**Comment by phadej** [comment]
amended changelog

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Comment by phadej** [comment]
Thanks!

**Pull request merged**